src/pages/single-video/single-video.js

```diff
@@ -1,13 +1,17 @@
-import React from "react";
+import { React, useEffect } from "react";
 import "./single-video.css";
 import { VideoCard } from "../../components/card/video-card";
 import { useVideos } from "../../context/video-context";
 import { useLocation } from "react-router";
 
 function SingleVideo() {
     const { videos, setVideos } = useVideos();
-    const location = useLocation();
-    const { video } = location.state;
+    const { pathname, state } = useLocation();
+    const { video } = state;
+
+    useEffect(() => {
+        window.scrollTo(0, 0);
+    }, [pathname]);
```

Comment:
Why is `scrollTo` added in useEffect? 
As anyways when page will be loaded it will always be scrolled to top

Reply:
if a user comes from the product/homepage, It will not scroll to top